Sources/MapboxDirections/RouteOptions.swift

```diff
@@ -129,7 +129,7 @@ open class RouteOptions: DirectionsOptions {
      
      The value of this property must be at least `DirectionsPriority.low` and at most `DirectionsPriority.high`. The default value of `DirectionsPriority.default` neither prefers nor avoids alleys, while a negative value between `DirectionsPriority.low` and `DirectionsPriority.default` avoids alleys, and a positive value between `DirectionsPriority.default` and `DirectionsPriority.high` prefers alleys. A value of 0.9 is suitable for pedestrians who are comfortable with walking down alleys.
```

Comment:
The comment needs an update. There is no "default" anymore. Also, it needs an explanation of what 'nil' means.

Reply:
Same for the other properties too.

Reply:
Thank you both! I will update the docs. Some examples can be updated after the next release. I can open a ticket for tail work.